assets/js/config/sites/gmail.js

```diff
@@ -42,5 +42,10 @@ module.exports = {
         }
       ]
     ]
+  },
+  injection: {
+    selector: 'div[role="main"] div[role="presentation"]',
```

Comment:
Actually this should be `'div[role="main"] div[role="complementary"]'` and not `'div[role="main"] div[role="presentation"]'`. (it was not working)
But even like that (prepending to the message) it's not working, so more investigation is needed.